cupy/cuda/function.pyx

```diff
@@ -119,12 +34,16 @@ cdef _launch(size_t func, Py_ssize_t grid0, int grid1, int grid2,
              args, Py_ssize_t shared_mem, size_t stream):
     cdef list pargs = []
     cdef vector.vector[void*] kargs
-    cdef CPointer cp
-    kargs.reserve(len(args))
-    for a in args:
+    cdef core.CPointer cp
+    cdef int i
+
+    kargs.resize(len(args))
+    for i, a in enumerate(args):
+        if a is None:
+            continue
```

Comment:
How about `kargs.push_back(nullptr)` here, instead of `kargs.resize()`? I think it is easier to understand.

Reply:
OK